proto/process/agent.proto

```diff
@@ -570,35 +578,103 @@ message ServiceSessionAffinityConfig {
+message NodeStatus {
+	// Key/Value of a resource type and a corresponding quantity as an int64.
+	map<string, int64> capacity = 1;
+	// Key/Value of a resource type and a corresponding quantity as an int64.
+	map<string, int64> allocatable = 2;
+	// https://github.com/kubernetes/kubernetes/blob/cb19b56831d54d1d31249949318ef0b07bf00df9/staging/src/k8s.io/api/core/v1/generated.proto#L2127
+	map<string,string> nodeAddresses = 3;
+	// node status displayed by "kubectl get node"
+	// mostly based on NodeCondition
+	// and https://github.com/kubernetes/kubernetes/blob/1e12d92a5179dbfeb455c79dbf9120c8536e5f9c/pkg/printers/internalversion/printers.go#L1410
+	string status = 4;
+	// https://github.com/kubernetes/kubernetes/blob/cb19b56831d54d1d31249949318ef0b07bf00df9/staging/src/k8s.io/api/core/v1/generated.proto#L2240
+	int32 daemonEndpoints = 5;
+	repeated ContainerImage images = 7;
+
+	// node system info reference https://github.com/kubernetes/kubernetes/blob/cb19b56831d54d1d31249949318ef0b07bf00df9/staging/src/k8s.io/api/core/v1/generated.proto#L2411
+	string kubeProxyVersion = 8;
+	string operatingSystem = 9;
+	string architecture = 10;
+	string kernelVersion = 11;
+	string osImage = 12;
+	string containerRuntimeVersion = 13;
+	string kubeletVersion = 14;
+	repeated NodeCondition conditions = 15;
+}
+
+// reference: https://github.com/kubernetes/kubernetes/blob/cb19b56831d54d1d31249949318ef0b07bf00df9/staging/src/k8s.io/api/core/v1/generated.proto#L2158
+message NodeCondition {
+	string type = 1;
+	string status = 2;
+	int64 lastTransitionTime = 3;
+	string reason = 4;
+	string message = 5;
+}
+
+// reference https://github.com/kubernetes/kubernetes/blob/cb19b56831d54d1d31249949318ef0b07bf00df9/staging/src/k8s.io/api/core/v1/generated.proto#L777
+message ContainerImage {
+	repeated string names = 1;
+	int64 sizeBytes = 2;
+}
+
+// reference https://github.com/kubernetes/kubernetes/blob/cb19b56831d54d1d31249949318ef0b07bf00df9/staging/src/k8s.io/api/core/v1/generated.proto#L4849
+message Taint {
+	string key = 1;
+	string value = 2;
+	string effect = 3;
+	int64 timeAdded = 4;
+}
+
 // reference https://github.com/kubernetes/kubernetes/blob/cb19b56831d54d1d31249949318ef0b07bf00df9/staging/src/k8s.io/api/core/v1/generated.proto#L4620
 message ServiceSpec {
-    repeated ServicePort ports = 1;
-    repeated LabelSelectorRequirement selectors = 2;
-    string clusterIP = 3;
-    string type = 4;
-    repeated string externalIPs = 5;
-    string sessionAffinity = 6;
-    string loadBalancerIP = 7;
-    repeated string loadBalancerSourceRanges = 8;
-    string externalName = 9;
-    string externalTrafficPolicy = 10;
-    int32 healthCheckNodePort = 11;
-    bool publishNotReadyAddresses = 12;
-    ServiceSessionAffinityConfig sessionAffinityConfig = 13;
-    string ipFamily = 14;
+	repeated ServicePort ports = 1;
```

Comment:
Sorry about adding those changes in this PR. 
In case this is too much I can move them to a separate PR.

Why: 
reformatted them to the same format as the other messages in this proto.